### PR TITLE
Add Ubuntu 22.04 support.

### PR DESCRIPTION
**Pull request by chrmatt**
## Purpose

The new node supports Ubuntu 22.04 in addition to 20.04. This pull requests adds the new Ubuntu version to the listed requirements.

## Changes

Added 22.04 to the Ubuntu requirements.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ You are now ready to select an instance on the cloud provider that matches the f
    * - Storage
      - 25 GB
    * - Operating system
-     - Ubuntu 20.04 x64
+     - Ubuntu 20.04 or 22.04 x64
```
Comment (abizjak) [inline source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst:44]: I'm not sure what `x64` is. Probably `x86-64` ?

**Review by abizjak — APPROVED** [review]

File: source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst (hunk 1 above)
Comment (chrmatt) [reply source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst:44]: Wikipedia says "x86-64 (also known as x64, x86_64, AMD64, and Intel 64)" ;) But that has been there and I only made minimal changes. I can replace by x86-64 if you prefer.

Comment (chrmatt) [reply source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst:44]: It's also weird that the 64 bit are only mentioned here and not in the other places. Maybe it's better to remove it (or add it to the other places if it is crucial).

Comment (abizjak) [reply source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst:44]: I am fine with removing it.

Comment (abizjak) [reply source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst:44]: Although our distribution does not work on ARM :P But that is true for windows as well.

Comment (chrmatt) [reply source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst]: But then the x86 is more relevant than the 64 part. ;) Anyway, it's gone.

Comment (abizjak) [reply source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst]: Well it also won't work on 32-bit machines :P

**Pull request merged**